tests/usecases/webhook/deployment.yml

```diff
@@ -12,8 +12,8 @@ application:
           inputs:
             # TODO(dliu): Please fix "repsotiory value" and see if we can change/remove username
```

Comment:
@daisy-ycguo Please remove the TODO comment and add a new comment telling user how to change the "username" and "accesstoken" and "repository".  For this, you can use fake example names.